### ISSUE TITLE
Assembly version wrong for ServerHost References added by NuGet

### ISSUE THREAD

**Issue by jthelin** [jthelin/ServerHost#3]
Adding the 1.0.2 package to new project using NuGet Manager adds a strange assembly version number (`25.0.0.0`) in the ServerHost reference XML.

``` xml
<Reference Include="ServerHost, Version=25.0.0.0, Culture=neutral, PublicKeyToken=4b8e72116a33a442, processorArchitecture=MSIL">
  <HintPath>..\..\packages\ServerHost.1.0.2\lib\net45\ServerHost.dll</HintPath>
</Reference>
```

Should really be `Version=1.0.0.0`

**Comment by jthelin** [comment]
Version is set correctly in source code.
https://github.com/jthelin/ServerHost/blob/master/ServerHost/Properties/AssemblyInfo.cs#L34
``` c#
[assembly: AssemblyVersion("1.0.0.0")]
[assembly: AssemblyFileVersion("1.0.0.0")]
```

**Comment by jthelin** [comment]
Fixed by 6d08ab4b6ba10035c5ed59790e9272a5c65d4ffb

Problem was old build config doing auto-version patching in AssemblyInfo.

After correcting build cofig and bumping package version to v1.0.3 to force rebuild, new package is back to `Version=1.0.0.0`

**Comment by jthelin** [comment]
Fixed in ServerHost release v1.0.3.
